helm/alfresco-identity-service/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: alfresco-identity-service
-version: 0.3.1
+version: 0.3.2
```

Comment:
This actually needs to be bumped to 0.3.3 now.

Reply:
Done